test/openAPI/features/wallet_link_status.feature

```diff
@@ -0,0 +1,36 @@
+@method=POST @endpoint=/linked-authorization/link-status
+Feature: The endpoint to checks the status of link code.
+
+  @smoke @unit @positive
+  Scenario: Successfully checks the status of link code smoke type test
+    Given Wants to check the status of link code
+    And The link code is generated
+    When Send POST /linked-authorization/link-status request with given X-XSRF-TOKEN header, transactionId, linkCode and requestTime
+    Then Receive a response from the /linked-authorization/link-status endpoint
+    And The /linked-authorization/link-status endpoint response should be returned in a timely manner 15000ms
+    And The /linked-authorization/link-status endpoint response should have status 200
+    And The /linked-authorization/link-status endpoint response should have content-type: application/json header
+    And The /linked-authorization/link-status endpoint response should match json schema with no errors
+    And The /linked-authorization/link-status response should contain transactionId property equals provided transactionId
+
+  @negative
+  Scenario: Not able to check the status of link code because of the invalid transactionId
+    Given Wants to check the status of link code
+    When Send POST /linked-authorization/link-status request with given X-XSRF-TOKEN header, invalid transactionId, linkCode and requestTime
```

Comment:
Can we have test scenarios for blank, random, completed transaction id separately ? All will result in invalid_transaction_id error code